devtools/ci/package.sh

```diff
@@ -38,7 +38,11 @@ if [ "${REL_PKG#*.}" = "tar.gz" ]; then
 else
   zip -r $PKG_NAME.zip $PKG_NAME
 fi
-if [ -n "${GPG_SIGNER:-}" ]; then
+if [ "$REL_QINIU" = "true" ]; then
+  qshell account "$QINIU_ACCESS_KEY" "$QINIU_SECRET_KEY" ckb
```

Comment:
Compress before upload will save a lot of time. And `7z` is better than `zip` and `gzip`.
- `make prod-with-debug`: `589 MiB`
- `zip ckb.zip ckb`: `145 MiB`
- `tar czf ckb.tgz ckb`: `145 MiB`
- `7z a -t7z ckb.7z ckb`: `99 MiB`